### PR TITLE
move AzureAD config from environment vars to the db

### PR DESCRIPTION
**Pull request by Schparky**
keeping the environment variables for use with social logins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ To add an organization using AzureAD authentication, create a database organizat
 that includes an auth_type of `azureadv2` and an auth_config like the following ... 
 
 ```
-{}
+{
+    "TenantID": "edf3cc03-7edf-4299-871a-940bc318789c",
```
Comment (fillup) [inline README.md]:
```suggestion
    "TenantID": "12345678-abcd-1234-871a-940bc318789c", 
    "ClientSecret": "nice and crazy complicated secret :-)", 
    "ApplicationID": "12345678-abcd-1234-92f6-fffe9f3dfc6d"
```

Comment (fillup) [reply README.md]: just trying to make these more obviously dummy data

File: README.md [hunk 2]
```diff
@@ -46,7 +46,11 @@ To add an organization using AzureAD authentication, create a database organizat
 that includes an auth_type of `azureadv2` and an auth_config like the following ... 
 
 ```
-{}
+{
+    "TenantID": "edf3cc03-7edf-4299-871a-940bc318789c", 
+    "ClientSecret": "XO;a?UV.6rEC(d+sDhZ6:8*7%GqjVuw?", 
+    "ApplicationID": "4a01326d-5130-4326-92f6-fffe9f3dfc6d"
+}
```
Comment (Baggerone) [inline README.md]: I guess the two lines below this should deleted or at least modified (i.e. no env vars for azure).

File: README.md (hunk 1 above)
Comment (fillup) [reply README.md]: not sure if that multi-line suggestion will be incorporated properly, just heads up

File: README.md (hunk 2 above)
Comment (Schparky) [reply README.md]: I started to modify them, but wasn't sure what you were going to do with Microsoft social login. Would you like me to remove them until that's working?

Comment (Baggerone) [reply README.md]: I think we will use a different auth type for theMicrosoft social login, which will have its own instructions.

**Pull request merged**